src/styles/card.css

```
@@ -0,0 +1,86 @@
+.card {
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 19px 19px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__info {
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 4px;
+  width: 166px;
```

```suggestion
  width: 100%;
```
Елемент має бути на всю ширина. не потрібно це вказувати явно

---

fix Vitaliy's comment
width should be 100%